include/seastar/json/json_elements.hh

```diff
@@ -194,6 +194,9 @@ public:
 
 class jsonable {
 public:
+    jsonable() = default;
+    jsonable(const jsonable&) = default;
+    jsonable& operator=(const jsonable&) = default;
```

Comment:
I know nothing about jsonable thing, but I'd try to be on the safe-side and start with allowing it to be move-able and delete the copy-constructor (explicitly or implicitly) WDYT?

Reply:
good idea! but tried. failed. it is actually used and required
```
/home/kefu/.local/bin/clang++ -DBOOST_NO_CXX98_FUNCTION_BASE -DFMT_SHARED -DSEASTAR_API_LEVEL=7 -DSEASTAR_BUILD_SHARED_LIBS -DSEASTAR_DEBUG -DSEASTAR_DEBUG_SHARED_PTR -DSEASTAR_DEFAULT_ALLOCATOR -DSEASTAR_DEFERRED_ACTION_REQUIRE_NOEXCEPT -DSEASTAR_HAS_MEMBARRIER -DSEASTAR_HAVE_ASAN_FIBER_SUPPORT -DSEASTAR_HAVE_HWLOC -DSEASTAR_HAVE_NUMA -DSEASTAR_HAVE_SYSTEMTAP_SDT -DSEASTAR_HAVE_URING -DSEASTAR_LOGGER_TYPE_STDOUT -DSEASTAR_SCHEDULING_GROUPS_COUNT=16 -DSEASTAR_SHUFFLE_TASK_QUEUE -DSEASTAR_SSTRING -DSEASTAR_TYPE_ERASE_MORE -I/home/kefu/dev/seastar/apps/httpd -I/home/kefu/dev/seastar/build/debug/apps/httpd -I/home/kefu/dev/seastar/include -I/home/kefu/dev/seastar/build/debug/gen/include -g -std=gnu++20 -UNDEBUG -Wall -Werror -Wimplicit-fallthrough -Wdeprecated -gz -U_FORTIFY_SOURCE -Wno-error=unused-result "-Wno-error=#warnings" -fstack-clash-protection -fsanitize=address -fsanitize=undefined -fno-sanitize=vptr -MD -MT apps/httpd/CMakeFiles/app_httpd.dir/demo.json.cc.o -MF apps/httpd/CMakeFiles/app_httpd.dir/demo.json.cc.o.d -o apps/httpd/CMakeFiles/app_httpd.dir/demo.json.cc.o -c /home/kefu/dev/seastar/build/debug/apps/httpd/demo.json.cc
In file included from /home/kefu/dev/seastar/build/debug/apps/httpd/demo.json.cc:1:
In file included from /home/kefu/dev/seastar/build/debug/apps/httpd/demo.json.hh:19:
/usr/include/boost/range/irange.hpp:211:39: error: call to implicitly-deleted copy constructor of 'seastar::httpd::demo_json::my_object::enum_var_wrapper'
        return integer_range<Integer>(first, last);
                                      ^~~~~
/home/kefu/dev/seastar/build/debug/apps/httpd/demo.json.hh:97:27: note: in instantiation of function template specialization 'boost::irange<seastar::httpd::demo_json::my_object::enum_var_wrapper>' requested here
            return boost::irange(begin(), end());
                          ^
/home/kefu/dev/seastar/build/debug/apps/httpd/demo.json.hh:45:31: note: copy constructor of 'enum_var_wrapper' is implicitly deleted because base class 'json::jsonable' has a deleted copy constructor
    struct enum_var_wrapper : public json::jsonable {
                              ^
/home/kefu/dev/seastar/include/seastar/json/json_elements.hh:198:5: note: copy constructor is implicitly deleted because 'jsonable' has a user-declared move constructor
    jsonable(jsonable&&) = default;
    ^
/usr/include/boost/range/irange.hpp:186:31: note: passing argument to parameter 'first' here
        integer_range(Integer first, Integer last)
                              ^
1 error generated.
```